frontend/src/lib/doc-util.tsx

```diff
@@ -55,9 +55,9 @@ export const DOC_LINKS = {
   HYPERSHIFT_INTRO: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-intro`,
   HYPERSHIFT_DEPLOY_AWS: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-aws`,
   HYPERSHIFT_MANAGE_KUBEVIRT: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-kubevirt`,
-  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#hosted-enable-feature-aws`,
+  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#hosting-service-cluster-configure-aws`,
```

Comment:
@fxiang1 The doc anchor we were using only existed until ACM 2.8; can you confirm if this is the best one to use now?

Reply:
@KevinFCormier  Good catch! Looks like it should be this one now:
https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html-single/clusters/index#enable-or-disable-hosted-control-planes

Can you change the anchor to this?
#enable-or-disable-hosted-control-planes

Reply:
No problem. Updated.

Reply:
Thanks!